manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
wondering... how autoproj checks the linux distro? 
I use linux mint (that is ubuntu based), but could this be the root of the problem?